core/compiler_target.h

```diff
@@ -5,6 +5,7 @@
 
 #include "backends/p4tools/common/compiler/compiler_target.h"
 #include "backends/p4tools/modules/flay/control_plane/control_plane_item.h"
+#include "control-plane/bfruntime.h"
```

Comment:
Remove this please.

Reply:
This is required when I compile recently. Do you have any idea what happened? (It is fine without the including before.)
```
make[2]: *** [backends/p4tools/modules/flay/CMakeFiles/libflay.dir/build.make:90: backends/p4tools/modules/flay/CMakeFiles/libflay.dir/flay.cpp.o] Error 1
In file included from /home/hank/p4c/backends/p4tools/modules/flay/core/program_info.h:7,
                 from /home/hank/p4c/backends/p4tools/modules/flay/core/externs.h:15,
                 from /home/hank/p4c/backends/p4tools/modules/flay/core/expression_resolver.h:7,
                 from /home/hank/p4c/backends/p4tools/modules/flay/core/expression_resolver.cpp:1:
/home/hank/p4c/backends/p4tools/modules/flay/core/compiler_target.h:19:9: error: ‘P4RuntimeAPI’ in namespace ‘P4’ does not name a type
   19 |     P4::P4RuntimeAPI p4runtimeApi;
      |         ^~~~~~~~~~~~
```

Reply:
There is something funky going on with the control-plane code recently. But bfruntime is the wrong include. https://github.com/p4lang/p4c/blob/bbc74ebd771b70f374f6f6419ee6acc38829fa59/control-plane/p4RuntimeSerializer.h#L53 should be one of them, or we just forward declare this API.